com.unity.render-pipelines.universal/Runtime/RenderTargetHandle.cs

```diff
@@ -5,25 +5,54 @@ namespace UnityEngine.Rendering.Universal
     [MovedFrom("UnityEngine.Rendering.LWRP")] public struct RenderTargetHandle
     {
         public int id { set; get; }
+        private RenderTargetIdentifier rtid { set; get; }
 
-        public static readonly RenderTargetHandle CameraTarget = new RenderTargetHandle {id = -1};
+        public static readonly RenderTargetHandle CameraTarget = new RenderTargetHandle {id = -1 };
+
+        public RenderTargetHandle(RenderTargetIdentifier renderTargetIdentifier)
+        {
+            id = -2;
```

Comment:
-2 again, what does it mean?

Reply:
-2 means RenderTargetHandle points to a valid renderTargetIdentifier.

RenderTargetHandle in URP has built-in -1 for CameraTarget. We use built-in -2 here for XR eye textures.